### PR TITLE
[all hosts] Refine hub page

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: docs/index.yml [hunk 1]
```diff
@@ -43,13 +43,13 @@ highlightedContent:
       title: Office Add-ins Beginner's guide
       url: overview/learning-path-beginner.md
     # Card 7
+    - itemType: sample
```
Comment (AlexJerabek) [inline docs/index.yml:46]: Is there a "Try it out" type?

File: docs/index.yml [hunk 2]
```diff
@@ -81,17 +84,17 @@ conceptualContent:
           text: Develop a full Excel add-in
         - url: tutorials/outlook-tutorial.md
           itemType: tutorial
-          text: Discover the Outlook add-in tutorial
-        - url: overview/patterns-and-practices-overview.md
-          itemType: sample
-          text: Use common Patterns and Practices
+          text: Develop a full Outlook add-in
         - url: /javascript/api/overview
```
Comment (AlexJerabek) [inline docs/index.yml:88]: Should we add Word here too?

**Review by AlexJerabek — APPROVED** [review]
Looks good! I left a couple thoughts for your consideration.

File: docs/index.yml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/index.yml:46]: unfortunately there isn't. sample seems to be the closest available option

File: docs/index.yml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/index.yml:91]: added

**Review by alison-mk — APPROVED** [review]

**Pull request merged**